__tests__/api.test.js

```diff
@@ -164,4 +165,28 @@ describe("API tests", () => {
             })
          })
     })
+    describe("GET /api/users", () => {
+        test("200: Return an array of all users", () => {
+            return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then(({body}) => {
+                expect(Array.isArray(body)).toBe(true)
+                expect(body.length).toBeGreaterThan(0)
+                body.forEach((user) => {
+                    expect(user).toHaveProperty("username")
+                    expect(user).toHaveProperty("name")
+                    expect(user).toHaveProperty("avatar_url")
+                })
```

Comment:
This is a good start, but body should not be the array of users. The body should be an object with a key of users which holds the array of users